frappe/public/images/ui-states/edit.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="13" height="13" viewBox="0 0 13 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this icon already loaded via `symbol-defs.svg`... you can use that one by

```html
<svg class="icon icon-sm">
	<use href="#icon-edit"></use>
</svg>
````
similarly for `right` icon as well